From: Marcel Holtmann <marcel@holtmann.org>
Date: Sun, 5 Jan 2014 00:44:34 -0800
Subject: hwdb: update

(cherry picked from commit 479e339e5534af4c800894a553010af91f5ec8f7)

Conflicts:
	hwdb/20-sdio-vendor-model.hwdb
---
 hwdb/20-OUI.hwdb              |  97 +++++++++++++++++++++++++-
 hwdb/20-pci-vendor-model.hwdb | 154 ++++++++++++++++++++++++++++++++++++++----
 hwdb/20-usb-vendor-model.hwdb |   6 ++
 3 files changed, 241 insertions(+), 16 deletions(-)

diff --git a/hwdb/20-OUI.hwdb b/hwdb/20-OUI.hwdb
index 92b96ae..a51a00a 100644
--- a/hwdb/20-OUI.hwdb
+++ b/hwdb/20-OUI.hwdb
@@ -13627,6 +13627,12 @@ OUI:40D8551E2*
 OUI:40D8551E3*
  ID_OUI_FROM_DATABASE=Mega Electronics Ltd
 
+OUI:40D8551E4*
+ ID_OUI_FROM_DATABASE=STEK Ltd
+
+OUI:40D855EE6*
+ ID_OUI_FROM_DATABASE=Narinet, Inc.
+
 OUI:000000*
  ID_OUI_FROM_DATABASE=XEROX CORPORATION
 
@@ -35222,7 +35228,7 @@ OUI:001C4C*
  ID_OUI_FROM_DATABASE=Petrotest Instruments
 
 OUI:001C4D*
- ID_OUI_FROM_DATABASE=Zeemote Technology Inc. (part of Aplix).
+ ID_OUI_FROM_DATABASE=Aplix IP Holdings Corporation
 
 OUI:001C4E*
  ID_OUI_FROM_DATABASE=TASA International Limited
@@ -52606,6 +52612,9 @@ OUI:08D42B*
 OUI:08D5C0*
  ID_OUI_FROM_DATABASE=Seers Technology Co., Ltd
 
+OUI:08D833*
+ ID_OUI_FROM_DATABASE=Shenzhen RF Technology Co,.Ltd
+
 OUI:08E5DA*
  ID_OUI_FROM_DATABASE=NANJING FUJITSU COMPUTER PRODUCTS CO.,LTD.
 
@@ -53308,6 +53317,9 @@ OUI:1423D7*
 OUI:142BD2*
  ID_OUI_FROM_DATABASE=Armtel Ltd.
 
+OUI:142D27*
+ ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
+
 OUI:142D8B*
  ID_OUI_FROM_DATABASE=Incipio Technologies, Inc
 
@@ -53761,6 +53773,9 @@ OUI:18C451*
 OUI:18C8E7*
  ID_OUI_FROM_DATABASE=Shenzhen Hualistone Technology Co.,Ltd
 
+OUI:18CC23*
+ ID_OUI_FROM_DATABASE=Philio Technology Corporation
+
 OUI:18D071*
  ID_OUI_FROM_DATABASE=DASAN SMC, Inc.
 
@@ -54688,6 +54703,9 @@ OUI:2838CF*
 OUI:2839E7*
  ID_OUI_FROM_DATABASE=Preceno Technology Pte.Ltd.
 
+OUI:283B96*
+ ID_OUI_FROM_DATABASE=Cool Control LTD
+
 OUI:283CE4*
  ID_OUI_FROM_DATABASE=Huawei Technologies Co., Ltd
 
@@ -54907,6 +54925,9 @@ OUI:28E14C*
 OUI:28E297*
  ID_OUI_FROM_DATABASE=Shanghai InfoTM Microelectronics Co.,Ltd.
 
+OUI:28E347*
+ ID_OUI_FROM_DATABASE=Liteon Technology Corporation
+
 OUI:28E608*
  ID_OUI_FROM_DATABASE=Tokheim
 
@@ -55717,6 +55738,9 @@ OUI:34E0CF*
 OUI:34E0D7*
  ID_OUI_FROM_DATABASE=DONGGUAN QISHENG ELECTRONICS INDUSTRIAL CO., LTD
 
+OUI:34E2FD*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:34EF44*
  ID_OUI_FROM_DATABASE=2Wire
 
@@ -56023,6 +56047,9 @@ OUI:3C1040*
 OUI:3C106F*
  ID_OUI_FROM_DATABASE=ALBAHITH TECHNOLOGIES
 
+OUI:3C15C2*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:3C15EA*
  ID_OUI_FROM_DATABASE=TESCOM CO., LTD.
 
@@ -56227,6 +56254,9 @@ OUI:3CC99E*
 OUI:3CCA87*
  ID_OUI_FROM_DATABASE=Iders Incorporated
 
+OUI:3CCD93*
+ ID_OUI_FROM_DATABASE=LG ELECTRONICS INC
+
 OUI:3CCE73*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
@@ -56408,7 +56438,7 @@ OUI:40667A*
  ID_OUI_FROM_DATABASE=mediola - connected living AG
 
 OUI:406826*
- ID_OUI_FROM_DATABASE=Thales Optronics Limited
+ ID_OUI_FROM_DATABASE=Thales UK Limited
 
 OUI:406AAB*
  ID_OUI_FROM_DATABASE=RIM
@@ -57436,6 +57466,9 @@ OUI:5048EB*
 OUI:504A5E*
  ID_OUI_FROM_DATABASE=Masimo Corporation
 
+OUI:504A6E*
+ ID_OUI_FROM_DATABASE=NETGEAR INC.,
+
 OUI:504F94*
  ID_OUI_FROM_DATABASE=Loxone Electronics GmbH
 
@@ -58480,6 +58513,9 @@ OUI:6045BD*
 OUI:604616*
  ID_OUI_FROM_DATABASE=XIAMEN VANN INTELLIGENT CO., LTD
 
+OUI:6047D4*
+ ID_OUI_FROM_DATABASE=FORICS Electronic Technology Co., Ltd.
+
 OUI:604A1C*
  ID_OUI_FROM_DATABASE=SUYIN Corporation
 
@@ -58630,6 +58666,9 @@ OUI:60D30A*
 OUI:60D819*
  ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
 
+OUI:60D9C7*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:60DA23*
  ID_OUI_FROM_DATABASE=Estech Co.,Ltd
 
@@ -58849,6 +58888,9 @@ OUI:6473E2*
 OUI:647657*
  ID_OUI_FROM_DATABASE=Innovative Security Designs
 
+OUI:6476BA*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:647791*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
@@ -58882,6 +58924,9 @@ OUI:6487D7*
 OUI:6488FF*
  ID_OUI_FROM_DATABASE=Sichuan Changhong Electric Ltd.
 
+OUI:648D9E*
+ ID_OUI_FROM_DATABASE=IVT Electronic Co.,Ltd
+
 OUI:64995D*
  ID_OUI_FROM_DATABASE=LGE
 
@@ -59161,6 +59206,9 @@ OUI:687924*
 OUI:6879ED*
  ID_OUI_FROM_DATABASE=SHARP Corporation
 
+OUI:687CC8*
+ ID_OUI_FROM_DATABASE=Measurement Systems S. de R.L.
+
 OUI:687CD5*
  ID_OUI_FROM_DATABASE=Y Soft Corporation, a.s.
 
@@ -59425,6 +59473,9 @@ OUI:6C6F18*
 OUI:6C7039*
  ID_OUI_FROM_DATABASE=Novar GmbH
 
+OUI:6C709F*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:6C71D9*
  ID_OUI_FROM_DATABASE=AzureWave Technologies, Inc
 
@@ -59779,6 +59830,9 @@ OUI:708B78*
 OUI:708D09*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
+OUI:709383*
+ ID_OUI_FROM_DATABASE=Intelligent Optical Network High Tech CO.,LTD.
+
 OUI:7093F8*
  ID_OUI_FROM_DATABASE=Space Monkey, Inc.
 
@@ -60166,6 +60220,9 @@ OUI:74F612*
 OUI:74F726*
  ID_OUI_FROM_DATABASE=Neuron Robotics
 
+OUI:74F85D*
+ ID_OUI_FROM_DATABASE=Berkeley Nucleonics Corp
+
 OUI:74FDA0*
  ID_OUI_FROM_DATABASE=Compupal (Group) Corporation
 
@@ -60862,6 +60919,9 @@ OUI:7CFADF*
 OUI:7CFE28*
  ID_OUI_FROM_DATABASE=Salutron Inc.
 
+OUI:7CFF62*
+ ID_OUI_FROM_DATABASE=Huizhou Super Electron Technology Co.,Ltd.
+
 OUI:80000B*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
@@ -61105,6 +61165,9 @@ OUI:80D019*
 OUI:80D18B*
  ID_OUI_FROM_DATABASE=Hangzhou I'converge Technology Co.,Ltd
 
+OUI:80D433*
+ ID_OUI_FROM_DATABASE=LzLabs GmbH
+
 OUI:80D733*
  ID_OUI_FROM_DATABASE=QSR Automations, Inc.
 
@@ -61336,6 +61399,9 @@ OUI:84ACA4*
 OUI:84AF1F*
  ID_OUI_FROM_DATABASE=Beat System Service Co,. Ltd.
 
+OUI:84B59C*
+ ID_OUI_FROM_DATABASE=Juniper networks
+
 OUI:84C2E4*
  ID_OUI_FROM_DATABASE=Jiangsu Qinheng Co., Ltd.
 
@@ -61402,6 +61468,9 @@ OUI:880355*
 OUI:880905*
  ID_OUI_FROM_DATABASE=MTMCommunications
 
+OUI:880FB6*
+ ID_OUI_FROM_DATABASE=Jabil Circuits India Pvt Ltd,-EHTP unit
+
 OUI:881036*
  ID_OUI_FROM_DATABASE=Panodic(ShenZhen) Electronics Limted
 
@@ -62605,6 +62674,9 @@ OUI:983000*
 OUI:983071*
  ID_OUI_FROM_DATABASE=DAIKYUNG VASCOM
 
+OUI:98349D*
+ ID_OUI_FROM_DATABASE=Krauss Maffei Technologies GmbH
+
 OUI:983571*
  ID_OUI_FROM_DATABASE=Sub10 Systems Ltd
 
@@ -63061,6 +63133,9 @@ OUI:9CCAD9*
 OUI:9CCD82*
  ID_OUI_FROM_DATABASE=CHENG UEI PRECISION INDUSTRY CO.,LTD
 
+OUI:9CD21E*
+ ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
+
 OUI:9CD24B*
  ID_OUI_FROM_DATABASE=zte corporation
 
@@ -63358,6 +63433,9 @@ OUI:A0CEC8*
 OUI:A0CF5B*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
+OUI:A0D12A*
+ ID_OUI_FROM_DATABASE=AXPRO Technology Inc.
+
 OUI:A0D3C1*
  ID_OUI_FROM_DATABASE=Hewlett Packard
 
@@ -64366,6 +64444,9 @@ OUI:B05B1F*
 OUI:B05CE5*
  ID_OUI_FROM_DATABASE=Nokia Corporation
 
+OUI:B061C7*
+ ID_OUI_FROM_DATABASE=Ericsson-LG Enterprise
+
 OUI:B06563*
  ID_OUI_FROM_DATABASE=Shanghai Railway Communication Factory
 
@@ -64684,6 +64765,9 @@ OUI:B46293*
 OUI:B462AD*
  ID_OUI_FROM_DATABASE=raytest GmbH
 
+OUI:B46698*
+ ID_OUI_FROM_DATABASE=Zealabs srl
+
 OUI:B467E9*
  ID_OUI_FROM_DATABASE=Qingdao GoerTek Technology Co., Ltd.
 
@@ -65590,6 +65674,9 @@ OUI:C098E5*
 OUI:C09C92*
  ID_OUI_FROM_DATABASE=COBY
 
+OUI:C09D26*
+ ID_OUI_FROM_DATABASE=Topicon HK Lmd.
+
 OUI:C09F42*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -65833,6 +65920,9 @@ OUI:C45DD8*
 OUI:C46044*
  ID_OUI_FROM_DATABASE=Everex Electronics Limited
 
+OUI:C4626B*
+ ID_OUI_FROM_DATABASE=ZPT Vigantice
+
 OUI:C462EA*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
@@ -68605,6 +68695,9 @@ OUI:EC14F6*
 OUI:EC172F*
  ID_OUI_FROM_DATABASE=TP-LINK TECHNOLOGIES CO., LTD.
 
+OUI:EC1766*
+ ID_OUI_FROM_DATABASE=Research Centre Module
+
 OUI:EC1A59*
  ID_OUI_FROM_DATABASE=Belkin International Inc.
 
diff --git a/hwdb/20-pci-vendor-model.hwdb b/hwdb/20-pci-vendor-model.hwdb
index 6142dc8..7f833e3 100644
--- a/hwdb/20-pci-vendor-model.hwdb
+++ b/hwdb/20-pci-vendor-model.hwdb
@@ -4166,6 +4166,9 @@ pci:v00001002d00006600*
 pci:v00001002d00006601*
  ID_MODEL_FROM_DATABASE=Mars [Radeon HD 8730M]
 
+pci:v00001002d00006601sv0000103Csd00002100*
+ ID_MODEL_FROM_DATABASE=FirePro M4100
+
 pci:v00001002d00006602*
  ID_MODEL_FROM_DATABASE=Mars
 
@@ -4185,10 +4188,22 @@ pci:v00001002d00006607*
  ID_MODEL_FROM_DATABASE=Mars LE [Radeon HD 8530M]
 
 pci:v00001002d00006610*
- ID_MODEL_FROM_DATABASE=Oland XT [Radeon HD 8670/R7 250]
+ ID_MODEL_FROM_DATABASE=Oland XT [Radeon HD 8670 / R7 250]
 
 pci:v00001002d00006611*
- ID_MODEL_FROM_DATABASE=Oland [Radeon HD 8570]
+ ID_MODEL_FROM_DATABASE=Oland [Radeon HD 8570 / R7 240 OEM]
+
+pci:v00001002d00006611sv00001028sd0000210B*
+ ID_MODEL_FROM_DATABASE=Radeon R5 240 OEM
+
+pci:v00001002d00006611sv0000174Bsd00004248*
+ ID_MODEL_FROM_DATABASE=Radeon R7 240 OEM
+
+pci:v00001002d00006611sv0000174Bsd0000A240*
+ ID_MODEL_FROM_DATABASE=Radeon R7 240 OEM
+
+pci:v00001002d00006611sv00001B0Asd000090D3*
+ ID_MODEL_FROM_DATABASE=Radeon R7 240 OEM
 
 pci:v00001002d00006613*
  ID_MODEL_FROM_DATABASE=Oland PRO [Radeon R7 240]
@@ -4224,7 +4239,7 @@ pci:v00001002d00006658*
  ID_MODEL_FROM_DATABASE=Bonaire XTX [Radeon R7 260X]
 
 pci:v00001002d0000665C*
- ID_MODEL_FROM_DATABASE=Bonaire XT [Radeon HD 7790/8770]
+ ID_MODEL_FROM_DATABASE=Bonaire XT [Radeon HD 7790/8770 / R9 260 OEM]
 
 pci:v00001002d0000665Csv00001043sd00000452*
  ID_MODEL_FROM_DATABASE=Radeon HD 7790 DirectCU II OC
@@ -4235,6 +4250,12 @@ pci:v00001002d0000665Csv00001462sd00002930*
 pci:v00001002d0000665Csv00001462sd00002932*
  ID_MODEL_FROM_DATABASE=Radeon HD 8770
 
+pci:v00001002d0000665Csv00001462sd00002934*
+ ID_MODEL_FROM_DATABASE=Radeon R9 260 OEM
+
+pci:v00001002d0000665Csv0000148Csd00009260*
+ ID_MODEL_FROM_DATABASE=Radeon R9 260 OEM
+
 pci:v00001002d0000665Csv00001682sd00003310*
  ID_MODEL_FROM_DATABASE=Radeon HD 7790 Black Edition 2 GB
 
@@ -4817,9 +4838,21 @@ pci:v00001002d00006742sv0000148Csd00006570*
 pci:v00001002d00006742sv00001682sd00006570*
  ID_MODEL_FROM_DATABASE=Turks [Radeon HD 6570]
 
+pci:v00001002d00006742sv0000174Bsd00005570*
+ ID_MODEL_FROM_DATABASE=Turks [Radeon HD 5570]
+
 pci:v00001002d00006742sv0000174Bsd00006570*
  ID_MODEL_FROM_DATABASE=Turks [Radeon HD 6570]
 
+pci:v00001002d00006742sv0000174Bsd00007570*
+ ID_MODEL_FROM_DATABASE=Turks [Radeon HD 7570]
+
+pci:v00001002d00006742sv0000174Bsd00008510*
+ ID_MODEL_FROM_DATABASE=Turks [Radeon HD 8510]
+
+pci:v00001002d00006742sv0000174Bsd00008570*
+ ID_MODEL_FROM_DATABASE=Turks [Radeon HD 8570]
+
 pci:v00001002d00006742sv00001787sd00006570*
  ID_MODEL_FROM_DATABASE=Turks [Radeon HD 6570]
 
@@ -5541,13 +5574,13 @@ pci:v00001002d00006770sv000017AAsd00003658*
  ID_MODEL_FROM_DATABASE=Radeon HD 7470A
 
 pci:v00001002d00006771*
- ID_MODEL_FROM_DATABASE=Caicos XTX [Radeon HD 8490]
+ ID_MODEL_FROM_DATABASE=Caicos XTX [Radeon HD 8490 / R5 235X OEM]
 
 pci:v00001002d00006772*
  ID_MODEL_FROM_DATABASE=Caicos [Radeon HD 7450A]
 
 pci:v00001002d00006778*
- ID_MODEL_FROM_DATABASE=Caicos XT [Radeon HD 7470/8470]
+ ID_MODEL_FROM_DATABASE=Caicos XT [Radeon HD 7470/8470 / R5 235 OEM]
 
 pci:v00001002d00006778sv00001019sd00000024*
  ID_MODEL_FROM_DATABASE=Radeon HD 7470
@@ -5580,7 +5613,7 @@ pci:v00001002d00006778sv0000174Bsd0000E145*
  ID_MODEL_FROM_DATABASE=Radeon HD 7470
 
 pci:v00001002d00006779*
- ID_MODEL_FROM_DATABASE=Caicos [Radeon HD 6450/7450/8450]
+ ID_MODEL_FROM_DATABASE=Caicos [Radeon HD 6450/7450/8450 / R5 230 OEM]
 
 pci:v00001002d00006779sv00001019sd00000016*
  ID_MODEL_FROM_DATABASE=Radeon HD 6450
@@ -5979,7 +6012,7 @@ pci:v00001002d00006831*
  ID_MODEL_FROM_DATABASE=Cape Verde [AMD Radeon HD 7700M Series]
 
 pci:v00001002d00006835*
- ID_MODEL_FROM_DATABASE=Cape Verde PRX [Radeon R7 260]
+ ID_MODEL_FROM_DATABASE=Cape Verde PRX [Radeon R9 255 OEM]
 
 pci:v00001002d00006837*
  ID_MODEL_FROM_DATABASE=Cape Verde LE [Radeon HD 7730/8730]
@@ -8967,7 +9000,7 @@ pci:v00001002d0000999C*
  ID_MODEL_FROM_DATABASE=Richland
 
 pci:v00001002d0000999D*
- ID_MODEL_FROM_DATABASE=Richland
+ ID_MODEL_FROM_DATABASE=Richland [Radeon HD 8550D]
 
 pci:v00001002d000099A0*
  ID_MODEL_FROM_DATABASE=Trinity [Radeon HD 7520G]
@@ -17663,6 +17696,9 @@ pci:v0000109Ed0000036Esv00001461sd00000761*
 pci:v0000109Ed0000036Esv00001461sd00000771*
  ID_MODEL_FROM_DATABASE=AverMedia AVerTV DVB-T 771
 
+pci:v0000109Ed0000036Esv00001464sd0000AA00*
+ ID_MODEL_FROM_DATABASE=iTuner Spectra8
+
 pci:v0000109Ed0000036Esv000014F1sd00000001*
  ID_MODEL_FROM_DATABASE=Bt878 Mediastream Controller NTSC
 
@@ -27503,6 +27539,9 @@ pci:v000010ECd00005209*
 pci:v000010ECd00005227*
  ID_MODEL_FROM_DATABASE=RTS5227 PCI Express Card Reader
 
+pci:v000010ECd00005227sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v000010ECd00005229*
  ID_MODEL_FROM_DATABASE=RTS5229 PCI Express Card Reader
 
@@ -48170,9 +48209,18 @@ pci:v0000168Cd00000033*
 pci:v0000168Cd00000034*
  ID_MODEL_FROM_DATABASE=AR9462 Wireless Network Adapter
 
+pci:v0000168Cd00000034sv00001A56sd00002003*
+ ID_MODEL_FROM_DATABASE=Killer Wireless-N 1202 Half-size Mini PCIe Card
+
 pci:v0000168Cd00000036*
  ID_MODEL_FROM_DATABASE=QCA9565 / AR9565 Wireless Network Adapter
 
+pci:v0000168Cd00000037*
+ ID_MODEL_FROM_DATABASE=AR9485 Wireless Network Adapter
+
+pci:v0000168Cd00000037sv00001A3Bsd00002100*
+ ID_MODEL_FROM_DATABASE=AW-NB100H 802.11n Wireless Mini PCIe Card
+
 pci:v0000168Cd0000003C*
  ID_MODEL_FROM_DATABASE=QCA988x 802.11ac Wireless Network Adapter
 
@@ -48191,6 +48239,9 @@ pci:v0000168Cd00009013*
 pci:v0000168Cd0000FF19*
  ID_MODEL_FROM_DATABASE=AR5006X Wireless Network Adapter
 
+pci:v0000168Cd0000FF1B*
+ ID_MODEL_FROM_DATABASE=AR2425 Wireless Network Adapter [AR5007EG 802.11bg]
+
 pci:v0000168Cd0000FF1C*
  ID_MODEL_FROM_DATABASE=AR5008 Wireless Network Adapter
 
@@ -49406,6 +49457,9 @@ pci:v00001814d00003298sv0000103Csd000018EC*
 pci:v00001814d00003592*
  ID_MODEL_FROM_DATABASE=RT3592 Wireless 802.11abgn 2T/2R PCIe
 
+pci:v00001814d0000359F*
+ ID_MODEL_FROM_DATABASE=RT3592 PCIe Wireless Network Adapter
+
 pci:v00001814d00005360*
  ID_MODEL_FROM_DATABASE=RT5360 Wireless 802.11n 1T/1R
 
@@ -49421,6 +49475,9 @@ pci:v00001814d00005390*
 pci:v00001814d00005390sv0000103Csd00001636*
  ID_MODEL_FROM_DATABASE=U98Z077.00 Half-size Mini PCIe Card
 
+pci:v00001814d00005392*
+ ID_MODEL_FROM_DATABASE=RT5392 PCIe Wireless Network Adapter
+
 pci:v00001814d0000539F*
  ID_MODEL_FROM_DATABASE=RT5390 [802.11 b/g/n 1T1R G-band PCI Express Single Chip]
 
@@ -50585,6 +50642,12 @@ pci:v00001957d00000087*
 pci:v00001957d000000B4*
  ID_MODEL_FROM_DATABASE=MPC8315E
 
+pci:v00001957d000000B6*
+ ID_MODEL_FROM_DATABASE=MPC8314E
+
+pci:v00001957d000000B6sv00001A56sd00001101*
+ ID_MODEL_FROM_DATABASE=Killer Xeno Pro Gigabit Ethernet Controller
+
 pci:v00001957d000000C2*
  ID_MODEL_FROM_DATABASE=MPC8379E
 
@@ -50669,6 +50732,12 @@ pci:v00001957d00007011*
 pci:v00001957d00007018*
  ID_MODEL_FROM_DATABASE=MPC8610
 
+pci:v00001957d0000C006*
+ ID_MODEL_FROM_DATABASE=MPC8308
+
+pci:v00001957d0000C006sv00001A56sd00001201*
+ ID_MODEL_FROM_DATABASE=Killer E2100 Gigabit Ethernet Controller
+
 pci:v00001958*
  ID_VENDOR_FROM_DATABASE=Faster Technology, LLC.
 
@@ -54164,6 +54233,9 @@ pci:v00008086d00000412*
 pci:v00008086d00000416*
  ID_MODEL_FROM_DATABASE=4th Gen Core Processor Integrated Graphics Controller
 
+pci:v00008086d00000416sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d0000041A*
  ID_MODEL_FROM_DATABASE=Xeon E3-1200 v3 Processor Integrated Graphics Controller
 
@@ -54680,9 +54752,15 @@ pci:v00008086d0000095Asv00008086sd00005000*
 pci:v00008086d0000095Asv00008086sd00005002*
  ID_MODEL_FROM_DATABASE=Wireless-N 7265
 
+pci:v00008086d0000095Asv00008086sd0000500A*
+ ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
+
 pci:v00008086d0000095Asv00008086sd00005010*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
+pci:v00008086d0000095Asv00008086sd00005012*
+ ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
+
 pci:v00008086d0000095Asv00008086sd00005020*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-N 7265
 
@@ -54695,6 +54773,9 @@ pci:v00008086d0000095Asv00008086sd00005090*
 pci:v00008086d0000095Asv00008086sd00005110*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
+pci:v00008086d0000095Asv00008086sd00005190*
+ ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
+
 pci:v00008086d0000095Asv00008086sd00005400*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
@@ -54707,24 +54788,30 @@ pci:v00008086d0000095Asv00008086sd00005420*
 pci:v00008086d0000095Asv00008086sd00005490*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
+pci:v00008086d0000095Asv00008086sd00005590*
+ ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
+
 pci:v00008086d0000095Asv00008086sd00009010*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
-pci:v00008086d0000095Asv00008086sd00009210*
+pci:v00008086d0000095Asv00008086sd00009110*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
-pci:v00008086d0000095Asv00008086sd00009410*
+pci:v00008086d0000095Asv00008086sd00009210*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
-pci:v00008086d0000095B*
- ID_MODEL_FROM_DATABASE=Wireless 7265
+pci:v00008086d0000095Asv00008086sd00009310*
+ ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
-pci:v00008086d0000095Bsv00008086sd0000500A*
+pci:v00008086d0000095Asv00008086sd00009410*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
-pci:v00008086d0000095Bsv00008086sd00005012*
+pci:v00008086d0000095Asv00008086sd00009510*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-AC 7265
 
+pci:v00008086d0000095B*
+ ID_MODEL_FROM_DATABASE=Wireless 7265
+
 pci:v00008086d0000095Bsv00008086sd00005200*
  ID_MODEL_FROM_DATABASE=Dual Band Wireless-N 7265
 
@@ -54860,6 +54947,9 @@ pci:v00008086d00000C01*
 pci:v00008086d00000C04*
  ID_MODEL_FROM_DATABASE=Xeon E3-1200 v3/4th Gen Core Processor DRAM Controller
 
+pci:v00008086d00000C04sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00000C05*
  ID_MODEL_FROM_DATABASE=Xeon E3-1200 v3/4th Gen Core Processor PCI Express x8 Controller
 
@@ -54872,6 +54962,9 @@ pci:v00008086d00000C09*
 pci:v00008086d00000C0C*
  ID_MODEL_FROM_DATABASE=Xeon E3-1200 v3/4th Gen Core Processor HD Audio Controller
 
+pci:v00008086d00000C0Csv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00000C46*
  ID_MODEL_FROM_DATABASE=Atom Processor S1200 PCI Express Root Port 1
 
@@ -57986,6 +58079,9 @@ pci:v00008086d00001539*
 pci:v00008086d0000153A*
  ID_MODEL_FROM_DATABASE=Ethernet Connection I217-LM
 
+pci:v00008086d0000153Asv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d0000153B*
  ID_MODEL_FROM_DATABASE=Ethernet Connection I217-V
 
@@ -69035,6 +69131,9 @@ pci:v00008086d00008C02*
 pci:v00008086d00008C03*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family 6-port SATA Controller 1 [AHCI mode]
 
+pci:v00008086d00008C03sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C04*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family SATA Controller 1 [RAID mode]
 
@@ -69062,12 +69161,18 @@ pci:v00008086d00008C0F*
 pci:v00008086d00008C10*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family PCI Express Root Port #1
 
+pci:v00008086d00008C10sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C11*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family PCI Express Root Port #1
 
 pci:v00008086d00008C12*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family PCI Express Root Port #2
 
+pci:v00008086d00008C12sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C13*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family PCI Express Root Port #2
 
@@ -69110,12 +69215,18 @@ pci:v00008086d00008C1F*
 pci:v00008086d00008C20*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset High Definition Audio Controller
 
+pci:v00008086d00008C20sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C21*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset High Definition Audio Controller
 
 pci:v00008086d00008C22*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family SMBus Controller
 
+pci:v00008086d00008C22sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C23*
  ID_MODEL_FROM_DATABASE=8 Series Chipset Family CHAP Counters
 
@@ -69125,12 +69236,21 @@ pci:v00008086d00008C24*
 pci:v00008086d00008C26*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family USB EHCI #1
 
+pci:v00008086d00008C26sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C2D*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family USB EHCI #2
 
+pci:v00008086d00008C2Dsv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C31*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family USB xHCI
 
+pci:v00008086d00008C31sv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C33*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family LAN Controller
 
@@ -69140,6 +69260,9 @@ pci:v00008086d00008C34*
 pci:v00008086d00008C3A*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family MEI Controller #1
 
+pci:v00008086d00008C3Asv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C3B*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family MEI Controller #2
 
@@ -69197,6 +69320,9 @@ pci:v00008086d00008C4E*
 pci:v00008086d00008C4F*
  ID_MODEL_FROM_DATABASE=QM87 Express LPC Controller
 
+pci:v00008086d00008C4Fsv000017AAsd0000220E*
+ ID_MODEL_FROM_DATABASE=ThinkPad T440p
+
 pci:v00008086d00008C50*
  ID_MODEL_FROM_DATABASE=B85 Express LPC Controller
 
diff --git a/hwdb/20-usb-vendor-model.hwdb b/hwdb/20-usb-vendor-model.hwdb
index 6663820..60dbcd2 100644
--- a/hwdb/20-usb-vendor-model.hwdb
+++ b/hwdb/20-usb-vendor-model.hwdb
@@ -46055,6 +46055,12 @@ usb:v1B5A*
 usb:v1B65*
  ID_VENDOR_FROM_DATABASE=The Hong Kong Standards and Testing Centre Ltd.
 
+usb:v1B71*
+ ID_VENDOR_FROM_DATABASE=Fushicai
+
+usb:v1B71p3002*
+ ID_MODEL_FROM_DATABASE=USBTV007 Video Grabber [EasyCAP]
+
 usb:v1B72*
  ID_VENDOR_FROM_DATABASE=ATERGI TECHNOLOGY CO., LTD.
 
